gatsby-config.js

```diff
@@ -112,6 +112,13 @@ module.exports = {
           }
         }
       }
+    },
+    {
+      resolve: 'gatsby-plugin-google-tagmanager',
+      options: {
+        id: 'GTM-K69NRND',
+        includeInDevelopment: false
```

Comment:
`false` is the default for this option in this version: https://www.npmjs.com/package/gatsby-plugin-google-tagmanager/v/2.11.0

We can remove this line and just give the plugin the GTM id